### PR TITLE
WIP: Add S.raw

### PR DESCRIPTION
**Pull request by aboutlo**
Related to #55 a draft.

Cover just `String` and `Object`. 

todo:
- [x] number
- [x] Boolean
- [x] Integer
- [x] Array
- [x] integration with AJV (e.g nullable) 
- ~Sanitaze input~
- ~Evaluate to support string as input~ ( postponed ) 
- [x] Coverage
- [ ] More recursion scenarios

### CODE REVIEW

File: src/FluentSchema.test.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 const S = require('./FluentSchema')
 
 describe('S', () => {
-  it('defined', () => {
-    expect(S).toBeDefined()
-  })
+  // it('defined', () => {
+  //   expect(S).toBeDefined()
+  // })
```
Comment (mcollina) [inline src/FluentSchema.test.js]: why?

Comment (aboutlo) [reply src/FluentSchema.test.js]: typo

**Comment by aboutlo** [comment]
> Good work!

🙏 Still, work to do 👻

**Comment by aboutlo** [comment]
@jnv and @firasdib can you test it and tell me if it covers your use cases? 🙏

**Comment by jnv** [comment]
@aboutlo Sure, let me check on Monday. Thank you!

**Comment by firasdib** [comment]
I'll try to find time this week, but it's looking good!

**Pull request merged**